dashboard/app/models/sections/section.rb

```diff
@@ -625,6 +626,8 @@ def update_code_review_expiration(enable_code_review)
     # Can't re-add someone who is already an instructor (or invited/declined)
     elsif si.present?
       raise ArgumentError.new('already invited')
+    elsif pl_section? && students.exists?(email: instructor.email)
+      raise ArgumentError.new('already a student')
```

Comment:
Will this also trigger a warning in the UI too?  That is, will users be notified that if they are trying to make someone in a section a co-teacher, they will need to kick that person out of the section first?

Reply:
I forgot that you could change FE and BE at the same time. I just added the error message that will get displayed.